client/state/themes/utils.js

```diff
@@ -211,6 +217,25 @@ export function isThemeFromWpcom( themeId ) {
 }
 
 /**
+ * Determine whether wpcom themes should be removed from
+ * a queried list of themes. For jetpack sites with the
+ * required capabilities, we hide wpcom themes from the
+ * list of locally installed themes.
+ *
+ * @param {Object} state Global state tree
+ * @param {number} siteId The Site ID
+ * @returns {boolean} true if wpcom themes should be removed
+ */
+export function shouldFilterWpcomThemes( state, siteId ) {
```

Comment:
I wouldn't mind the `config` in  `state/themes/selectors` -- I think we might have a bunch in there already? Actually, I'd prefer if `shouldFilterWpcomThemes` lived there, since it requires a `state` arg (that has been my criterion to discern selectors from utils -- if it needs `state`, it's a selector).